Update battery docs (#160)
husarion/panther_ros: panther_battery/README.md

### adc_node.py

Publishes battery state read from ADC unit. Available from Panther version 1.2. Voltage, current and temperature are smoothed out by moving average.

> 3

Reviewer: ```suggestion
Publishes battery state read from ADC unit. Available from Panther version 1.2. Voltage, current and temperature are smoothed out using a moving average.

```

- `/panther/driver/motor_controllers_state` [*panther_msgs/DriverState*]: current motor controllers' state and error flags.

> - `/panther/hardware/io_state` [*panther_msgs/IOState*]: checks if charger is connected. Later fuses the information with charging current.

Reviewer: ```suggestion
- `/panther/driver/motor_controllers_state` [*panther_msgs/DriverState*]: current motor controllers' state and error flags.
- `/panther/hardware/io_state` [*panther_msgs/IOState*]: checks if charger is connected. Later fuses the information with charging current.
```

#### Parameters 

- `~fatal_bat_temp` [*float*, default: **62.0**]: The temperature of the battery at which the battery health state is set to `POWER_SUPPLY_HEALTH_OVERHEAT`.

> 

Reviewer: ```suggestion
```

### roboteq_republisher_node.py

> Node publishing Panther battery state read from motor controllers. Used in Panther versions 1.06 and below. Voltage and current measurement are smoothed with moveing average. Current accounts only for motor controllers.

Reviewer: ```suggestion
Node publishing Panther battery state read from motor controllers. Used in Panther versions 1.06 and below. Voltage and current measurement are smoothed out using a moving average. Current accounts only for motor controllers.
```
